e2e/cypress.json

```diff
@@ -9,6 +9,7 @@
     "testFiles": "**/*_spec.js",
     "env": {
         "storybookUrl": "http://localhost:6006/",
-        "webhookBaseUrl": "http://localhost:3000"
+        "webhookBaseUrl": "http://localhost:3000",
+        "setChromeWebSecurity": false
```

Comment:
Should this be true by default so that all tests except SAML will have `chromeWebSecurity` enabled?  Yes, we can control this on CI but on local development, `chromeWebSecurity` will make it always disabled.

Reply:
sure, will do.